src/electron.ts

```diff
@@ -10,10 +10,17 @@ if (require("electron-squirrel-startup")) {
 
 let homeserverURL = ""
 let accessToken = ""
+let supportsAuthedMedia = false
 
-ipcMain.handle("mautrix:access-token-changed", (event, newDetails: AccessTokenChangedParams) => {
-	homeserverURL = newDetails.homeserverURL
+ipcMain.handle("mautrix:access-token-changed", async (event, newDetails: AccessTokenChangedParams) => {
+	homeserverURL = newDetails.homeserverURL.replace(/\/$/, "") // remove trailing slash
```

Comment:
This should probably be in the login process rather than here

Reply:
Moved.